src/dataframe/dataframe.jl

```diff
@@ -9,6 +9,7 @@ particularly a Vector, DataVector, or PooledDataVector.
 ```julia
 DataFrame(columns::Vector, names::Vector{Symbol})
 DataFrame(kwargs...)
+DataFrame(Pair{Symbol}...)
```

Comment:
Thanks for catching this. I've corrected the documentation. Note that looking at the documentation locally doesn't help much as the output format is still markdown.